civihr_default_theme/templates/contrib/views-slideshow-controls-text-next.tpl.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @file
+ * Views Slideshow: Template for text control - next.
+ *
+ * - $variables: Contains theme variables.
```

Comment:
It would be nice to let the IDE know these variables exist by putting an `@var` in front of them. You can use the Drupal coding style of having description on next line too, e.g.

```
@var $variables
  Contains theme variables.
```